frontend/src/scenes/surveys/surveysLogic.tsx

```diff
@@ -53,6 +68,11 @@ export const surveysLogic = kea<surveysLogicType>([
                 const updatedSurvey = await api.surveys.update(id, { ...updatePayload })
                 return values.surveys.map((survey) => (survey.id === id ? updatedSurvey : survey))
             },
+            duplicateSurvey: async (id) => {
+                const surveyToDuplicate = await api.surveys.get(id)
```

Comment:
for now let's not allow duplication via the survey list, only via an existing open survey - so we get rid of this two request problem.